### PR TITLE
Init

### PR DESCRIPTION
**Pull request by Kidswiss**
First iteration of the `metrics-server` component.

### CODE REVIEW

**Comment by Kidswiss** [comment]
Fixed all points.

File: class/defaults.yml [hunk 1]
```diff
@@ -1,3 +1,9 @@
 parameters:
   metrics_server:
     namespace: syn-metrics-server
+    charts:
+      metrics-server: '2.11.1'
+    images:
+      metrics_server:
+        image: docker.io/metrics-server-amd64
```
Comment (srueg) [inline class/defaults.yml:8]: Looks like a part of the URL is missing?

**Comment by srueg** [comment]
Please configure the access rights for this repo and add the sig-syn team as owners

**Comment by Kidswiss** [comment]
@srueg is it possible to configure github that these access rights are automagically applied to all new projects?

**Comment by srueg** [comment]
> @srueg is it possible to configure github that these access rights are automagically applied to all new projects?

Unfortunately not, no

**Pull request merged**